pages/projects/[id].tsx

```diff
@@ -28,12 +29,22 @@ const Projects = (): JSX.Element => {
   const [filters, setFilters] = useState<Filters>({});
   const stories = useSelector((state: State): Record<string, Story[]> => filterStories(state, id, filters));
 
-  const addFilter = (name: string, filter: Owner | Label): void => {
+  const addFilter = (name: string, filter: Owner | Label | Iteration): void => {
+    if (name === 'iterations') {
+      // @ts-ignore: we know iteration can only be Iteration type
```

Comment:
does anyone know how I can imply this using typescript?

Reply:
Not beautiful, but this is how I worked around it. 
If you like this approach better, you can leave a comment on the interface. 

![Screen Shot 2020-09-01 at 3 34 08 PM](https://user-images.githubusercontent.com/42581689/91897676-a196f400-ec68-11ea-84cb-0f823c4631c8.png)

Reply:
Yeah I think that just defers the work until I need to use the iteration :(

Reply:
@diegofigs I think this can be fixed with the type guard trick you just thought me right?